test/TestApps/PushNotificationsWin32App/main.cpp

```diff
@@ -1,12 +1,70 @@
 ﻿#include "pch.h"
 #include <iostream>
+#include <sstream>
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.Security.Cryptography.h>
+#include <winrt/Windows.Storage.Streams.h>
+#include <winrt/Windows.Web.Http.Headers.h>
+
 using namespace winrt;
+using namespace winrt::Windows::Storage;
+using namespace winrt::Windows::Storage::Streams;
 using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::ApplicationModel::Activation;
 using namespace winrt::Microsoft::ProjectReunion;
+using namespace Windows::Web::Http;
+
+std::wstring RequestChannel()
+{
+    return // Insert ChannelUri here.;
```

Comment:
Not sure what this method is for

Reply:
It's basically a method stub for Sharath to fill with the ProjectReunion component. Currently I am requesting a channel on our NotificationsTestApp and hardcoded that channel in the console app.

Reply:
Ah. Instead of doing that, can you simply call into our Public WinRT APIs here and return the string?